model/Core/Properties/originalNamespaces.md

```diff
@@ -0,0 +1,20 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# originalNamespaces
+
+## Summary
+
+Provides a NamespaceMap representing the NamspaceMap used for the original serialization of an SpdxDocument.
+
+## Description
+
+This field is used for providing a reliable mechanism in reproducing an original serialization for a given SPDX Document.
```

Comment:
instead of `reliable mechanism` for me it feels more like a `hint`.

We probably could also highlight that its support is optional? Especially that it can be omitted, even if some magic shortened IDs while serialization. So, one could serialize using some namespaceMap like features, but is not forced to store this information in that map? Might be helpful to simplify implementation. 


Especially if someone "mints" the SpdxDocument and later wants to serialize it. Otherwise, one would be forced to do the namespace choices before minting the element. And one would need to do the same choice for all serializations.

Reply:
It being just a hint, also resolves the concerns above.